staging/src/k8s.io/api/testdata/v1.25.0/apps.v1.StatefulSet.after_roundtrip.json

```diff
@@ -0,0 +1,1777 @@
+{
```

Comment:
All the "after_roundtrip" files are unexpected. It doesn't look like the new fields are here so unsure why it's getting generated.

Reply:
I manually generated those files after unit test on prow was unhappy but then I didn't had tombstone comment at that time. I have removed these generated files.